src/python/Utils/Utilities.py

```diff
@@ -33,6 +33,8 @@ def makeList(stringList):
     throws a ValueError if the input is not well formed.
     If the stringList is already of type list, then return it untouched.
     """
+    # NOTE: In case of a change to this function beware of side effects which
```

Comment:
Given that the `Utils` package is supposed to be agnostic of any WMCore-specific logic, I would suggest to remove this comment, please.